diff --git a/node_modules/@openzeppelin/contracts/access/Ownable.sol b/node_modules/@openzeppelin/contracts/access/Ownable.sol
index fb432ed..085fce3 100644
--- a/node_modules/@openzeppelin/contracts/access/Ownable.sol
+++ b/node_modules/@openzeppelin/contracts/access/Ownable.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 import "../utils/Context.sol";
 /**
diff --git a/node_modules/@openzeppelin/contracts/introspection/ERC165.sol b/node_modules/@openzeppelin/contracts/introspection/ERC165.sol
index 81ab7e5..89bb183 100644
--- a/node_modules/@openzeppelin/contracts/introspection/ERC165.sol
+++ b/node_modules/@openzeppelin/contracts/introspection/ERC165.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 import "./IERC165.sol";
 
diff --git a/node_modules/@openzeppelin/contracts/introspection/ERC165Checker.sol b/node_modules/@openzeppelin/contracts/introspection/ERC165Checker.sol
index 7250a76..9600ff4 100644
--- a/node_modules/@openzeppelin/contracts/introspection/ERC165Checker.sol
+++ b/node_modules/@openzeppelin/contracts/introspection/ERC165Checker.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.2 <0.8.0;
+pragma solidity >=0.6.2;
 
 /**
  * @dev Library used to query support of an interface declared via {IERC165}.
diff --git a/node_modules/@openzeppelin/contracts/introspection/IERC165.sol b/node_modules/@openzeppelin/contracts/introspection/IERC165.sol
index d474355..e1f3a24 100644
--- a/node_modules/@openzeppelin/contracts/introspection/IERC165.sol
+++ b/node_modules/@openzeppelin/contracts/introspection/IERC165.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 /**
  * @dev Interface of the ERC165 standard, as defined in the
diff --git a/node_modules/@openzeppelin/contracts/math/Math.sol b/node_modules/@openzeppelin/contracts/math/Math.sol
index a0c35b5..f1c8ea4 100644
--- a/node_modules/@openzeppelin/contracts/math/Math.sol
+++ b/node_modules/@openzeppelin/contracts/math/Math.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 /**
  * @dev Standard math utilities missing in the Solidity language.
diff --git a/node_modules/@openzeppelin/contracts/math/SafeMath.sol b/node_modules/@openzeppelin/contracts/math/SafeMath.sol
index 3e81f29..ab2bc36 100644
--- a/node_modules/@openzeppelin/contracts/math/SafeMath.sol
+++ b/node_modules/@openzeppelin/contracts/math/SafeMath.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 /**
  * @dev Wrappers over Solidity's arithmetic operations with added overflow
diff --git a/node_modules/@openzeppelin/contracts/token/ERC20/ERC20.sol b/node_modules/@openzeppelin/contracts/token/ERC20/ERC20.sol
index 1f3e400..a01f9c2 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC20/ERC20.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC20/ERC20.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 import "../../utils/Context.sol";
 import "./IERC20.sol";
diff --git a/node_modules/@openzeppelin/contracts/token/ERC20/IERC20.sol b/node_modules/@openzeppelin/contracts/token/ERC20/IERC20.sol
index 9b877d1..6b1cd3b 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC20/IERC20.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC20/IERC20.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 /**
  * @dev Interface of the ERC20 standard as defined in the EIP.
diff --git a/node_modules/@openzeppelin/contracts/token/ERC20/SafeERC20.sol b/node_modules/@openzeppelin/contracts/token/ERC20/SafeERC20.sol
index b051109..43ff527 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC20/SafeERC20.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC20/SafeERC20.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 import "./IERC20.sol";
 import "../../math/SafeMath.sol";
diff --git a/node_modules/@openzeppelin/contracts/token/ERC721/ERC721.sol b/node_modules/@openzeppelin/contracts/token/ERC721/ERC721.sol
index 7958fdf..2082dca 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC721/ERC721.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC721/ERC721.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 import "../../utils/Context.sol";
 import "./IERC721.sol";
diff --git a/node_modules/@openzeppelin/contracts/token/ERC721/ERC721Burnable.sol b/node_modules/@openzeppelin/contracts/token/ERC721/ERC721Burnable.sol
index 290118a..d9de44c 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC721/ERC721Burnable.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC721/ERC721Burnable.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 import "../../utils/Context.sol";
 import "./ERC721.sol";
diff --git a/node_modules/@openzeppelin/contracts/token/ERC721/ERC721Holder.sol b/node_modules/@openzeppelin/contracts/token/ERC721/ERC721Holder.sol
index c0910e2..a8bb07a 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC721/ERC721Holder.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC721/ERC721Holder.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 import "./IERC721Receiver.sol";
 
diff --git a/node_modules/@openzeppelin/contracts/token/ERC721/ERC721Pausable.sol b/node_modules/@openzeppelin/contracts/token/ERC721/ERC721Pausable.sol
index 9301e19..ec7e81b 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC721/ERC721Pausable.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC721/ERC721Pausable.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 import "./ERC721.sol";
 import "../../utils/Pausable.sol";
diff --git a/node_modules/@openzeppelin/contracts/token/ERC721/IERC721.sol b/node_modules/@openzeppelin/contracts/token/ERC721/IERC721.sol
index 787afde..3e0ae8f 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC721/IERC721.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC721/IERC721.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.2 <0.8.0;
+pragma solidity >=0.6.2;
 
 import "../../introspection/IERC165.sol";
 
diff --git a/node_modules/@openzeppelin/contracts/token/ERC721/IERC721Enumerable.sol b/node_modules/@openzeppelin/contracts/token/ERC721/IERC721Enumerable.sol
index cd08125..b21e780 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC721/IERC721Enumerable.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC721/IERC721Enumerable.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.2 <0.8.0;
+pragma solidity >=0.6.2;
 
 import "./IERC721.sol";
 
diff --git a/node_modules/@openzeppelin/contracts/token/ERC721/IERC721Metadata.sol b/node_modules/@openzeppelin/contracts/token/ERC721/IERC721Metadata.sol
index 9a64c87..701d3bc 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC721/IERC721Metadata.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC721/IERC721Metadata.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.2 <0.8.0;
+pragma solidity >=0.6.2;
 
 import "./IERC721.sol";
 
diff --git a/node_modules/@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol b/node_modules/@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol
index 85fd1e4..b199566 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 /**
  * @title ERC721 token receiver interface
diff --git a/node_modules/@openzeppelin/contracts/utils/Address.sol b/node_modules/@openzeppelin/contracts/utils/Address.sol
index 42a9dc1..7d0cae9 100644
--- a/node_modules/@openzeppelin/contracts/utils/Address.sol
+++ b/node_modules/@openzeppelin/contracts/utils/Address.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.2 <0.8.0;
+pragma solidity >=0.6.2;
 
 /**
  * @dev Collection of functions related to the address type
diff --git a/node_modules/@openzeppelin/contracts/utils/Context.sol b/node_modules/@openzeppelin/contracts/utils/Context.sol
index 3d07c4a..85f77fe 100644
--- a/node_modules/@openzeppelin/contracts/utils/Context.sol
+++ b/node_modules/@openzeppelin/contracts/utils/Context.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 /*
  * @dev Provides information about the current execution context, including the
@@ -13,7 +13,7 @@ pragma solidity >=0.6.0 <0.8.0;
  * This contract is only required for intermediate, library-like contracts.
  */
 abstract contract Context {
-    function _msgSender() internal view virtual returns (address payable) {
+    function _msgSender() internal view virtual returns (address) {
         return msg.sender;
     }
 
diff --git a/node_modules/@openzeppelin/contracts/utils/EnumerableMap.sol b/node_modules/@openzeppelin/contracts/utils/EnumerableMap.sol
index a3e054c..6a3ea96 100644
--- a/node_modules/@openzeppelin/contracts/utils/EnumerableMap.sol
+++ b/node_modules/@openzeppelin/contracts/utils/EnumerableMap.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 /**
  * @dev Library for managing an enumerable variant of Solidity's
diff --git a/node_modules/@openzeppelin/contracts/utils/EnumerableSet.sol b/node_modules/@openzeppelin/contracts/utils/EnumerableSet.sol
index 858312d..c11a833 100644
--- a/node_modules/@openzeppelin/contracts/utils/EnumerableSet.sol
+++ b/node_modules/@openzeppelin/contracts/utils/EnumerableSet.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 /**
  * @dev Library for managing
diff --git a/node_modules/@openzeppelin/contracts/utils/Strings.sol b/node_modules/@openzeppelin/contracts/utils/Strings.sol
index 5842448..03d1f3b 100644
--- a/node_modules/@openzeppelin/contracts/utils/Strings.sol
+++ b/node_modules/@openzeppelin/contracts/utils/Strings.sol
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0;
 
 /**
  * @dev String operations.
